packages/next/server/next-dev-server.ts

```diff
@@ -33,6 +33,16 @@ import HotReloader from './hot-reloader'
 import { findPageFile } from './lib/find-page-file'
 import { getNodeOptionsWithoutInspect } from './lib/utils'
 
+// NodeJs Version Validation
+var currentNodeVersion: Number = parseFloat(process.versions.node)
+
+// Throws an Error if NodeJs Version is less than 10.13.x
+if (currentNodeVersion < 10.13) {
+  throw new Error(
+    `The version of NodeJs you are using is lower than the minimum required version (10.13.x)`
```

Comment:
`The current version ${currentNodeVersion} of NodeJs you are using is lower than the minimum required. (10.13)

This will be better because it will let the user know his current version